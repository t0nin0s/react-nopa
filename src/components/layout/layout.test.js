import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Layout from './layout'

test('Layout component renders as expected', () => {
  const component = shallow(<Layout />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
