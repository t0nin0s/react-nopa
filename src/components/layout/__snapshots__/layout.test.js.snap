// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout component renders as expected 1`] = `
<div>
  <Header
    home={false}
  />
  <Footer />
</div>
`;
